extrinsic_matrix:
- - 0.04620152287289003
  - -0.9984331394530994
  - -0.03157032350251121
  - 0.022648147129484365
- - 0.006530230785538105
  - 0.03190527519252051
  - -0.9994695640692501
  - -0.010351822861637465
- - 0.9989107945007468
  - 0.04597085442665749
  - 0.007994071126131153
  - -0.308581868853531
- - 0.0
  - 0.0
  - 0.0
  - 1.0
